src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -424,7 +461,14 @@ export class StakingModuleGuardService {
   ): boolean {
     if (!firstState || !secondState) return false;
     if (firstState.depositRoot !== secondState.depositRoot) return false;
-    if (firstState.nonce !== secondState.nonce) return false;
+    if (
+      !this.isSameStakingModuleContractState(
```

Comment:
Tak, Anna, nu tbI chego? XD
TbI chto za stroki poluchaesh zp?)
^^^delete after reading^^^^
Just do `if (firstState.nonce != secondState.nonce)`

Reply:
i did it to highlight that it is staking module contract state change

Reply:
and to reuse it in another place

Reply:
But you are not comparing contracts. You compare nonces.
I would agree with you if you did
`compareContracts(contract_1, contract_2)`,
But in this version its overengineering to move one operation outside function

Reply:
fixed